xla/stream_executor/blas.h

```diff
@@ -222,6 +222,10 @@ class BlasSupport {
 
   virtual gpu::BlasLt *GetBlasLt() = 0;
 
+  // For tests only: sets *is_main_stream to true if the underlying Blas library 
+  // has stream 0 set as its current stream.
+  virtual bool IsMainStreamSet(bool *is_main_stream) = 0;
```

Comment:
Can you make this return a `absl::StatusOr<bool>` instead of using an out-parameter?

Reply:
And can you make this function `const`? (This probably requires marking the mutex in the implementations as `mutable`.

Reply:
yep, I have changed the interface accordingly: initially I just wanted to keep it similar to all other BlasSupport functions..